### DISCUSSION TITLE
transaction logs out of control

### DISCUSSION THREAD

**Discussion by stuartcporter** [olahallengren/sql-server-maintenance-solution#344] [Questions & Answers]
Transaction logs on mirrorred db just keep growing, now at 50gb backup size is 10k! 98% free space

Microsoft SQL Server 2014 (SP2-GDR) (KB4505217) - 12.0.5223.6 (X64) 
	May 26 2019 20:36:50 
	Copyright (c) Microsoft Corporation
	Standard Edition (64-bit) on Windows NT 6.3 <X64> (Build 9600: ) (Hypervisor)

Check the header of the stored procedure - this is no version info in the header , just today's date - i think it was installed 2017 ( person who installed is no longer here)

sqlcmd -E -S $(ESCAPE_SQUOTE(SRVR)) -d master -Q "EXECUTE [dbo].[DatabaseBackup] @Databases = 'USER_DATABASES', @Directory = N'E:\Backup', @BackupType = 'LOG', @Verify = 'Y', @CleanupTime =168, @CheckSum = 'Y', @LogToTable = 'Y', @Compress = 'Y'" -b

job completes - but trans log just keep growing

any we missing anything?

**Comment by olahallengren** [comment]
Could you check if the log backup is backed up?

You can see this in the dbo.CommandLog - table.
